CONTRIBUTING.md

```diff
@@ -85,6 +85,22 @@ To run the entire suite of tests:
 $ scripts/test.sh
 ```
 
+### Build Docs
+To build the docs run:
+```sh
+pip install -r docs/requirements.txt
+cd docs
+make html
+```
```

Comment:
One handy thing is `sphinx-livehtml` which you can install and then run `make livehtml`.

Reply:
ooh nice. ill add that then